src/main/kotlin/graphics/scenery/controls/behaviours/Enableable.kt

```diff
@@ -0,0 +1,8 @@
+package graphics.scenery.controls.behaviours
+
+/**
+ * The implementing class can be turned off and on.
+ */
+interface Enableable {
```

Comment:
I actually had to look it up - it's pronounced _Enableable_ 😂 (see https://en.wiktionary.org/wiki/enablable)